callflow/datastructures/supergraph.py

```diff
@@ -358,9 +358,9 @@ def create(
             self.paths[cs_idx] = [
                 _csidx(Sanitizer.from_htframe(_)) for _ in node.paths()[0]
             ]
-            self.callers[cs_idx] = [_csidx(_.frame.get("name")) for _ in node.parents]
+            self.callers[cs_idx] = [_csidx(Sanitizer.from_htframe(_.frame)) for _ in node.parents]
```

Comment:
I noticed that this was a bug for hatchet nodes that give {file: "X", line: Y}, we are sanitizing by appending them. For eg., sanitizing the above node will X:Y. This was being passed correct to `self.paths` but not for `self.callers` and `self.callees`.